com.napster.napster-bigscreen-electron.desktop

```
@@ -3,7 +3,7 @@ Encoding=UTF-8
 Name=Napster
 Comment=Stream music and videos using the Napster
 Type=Application
-Exec=
+Exec=/app/bin/start-napster-bigscreen-electron
```

```suggestion
Exec=start-napster-bigscreen-electron
```

Also because this is valid in non flatpak.